src/Auth0Client.ts

```diff
@@ -695,6 +696,28 @@ export default class Auth0Client {
     return !!user;
   }
 
+  /**
+   * ```js
+   * await auth0.buildLogoutUrl(options);
+   * ```
+   *
+   * Builds a `/v2/logout` URL for logout using the parameters provided as arguments.
```

Comment:
Let's tweak this a little:

```suggestion
   * Builds a URL to the logout endpoint using the parameters provided as arguments.
```

Reply:
It's worth mentioning I kept this from the existing `logout` description and is similarly formatted `buildAuthorizeUrl` should that want to be updated one day.